package.json

```diff
@@ -1,9 +1,8 @@
 {
   "name": "@transcend-io/penumbra",
-  "version": "4.7.2",
-  "description": "Crypto streams for the browser.",
-  "main": "build/penumbra.js",
-  "types": "build/src/index.d.ts",
+  "version": "4.7.3",
+  "description": "Encrypt/decrypt anything in the browser using streams on background threads.",
+  "main": "src/index.ts",
```

Comment:
Are you sure this is what we want to do?

The way i think this differs is:

## On Dev

- When you import `@transcend-io/penumbra`, it runs `build/penumbra.js` but maps the data to the types in `build/src/index.d.ts` so its best of both worlds

## This change

- When you import `@transcend-io/penumbra`, it tries to run `src/index.ts` which i think is an antipattern since you dont want to require ts, you want to require js